tests/FakeItEasy.Tests/Configuration/StartConfigurationTests.cs

```diff
@@ -1,223 +0,0 @@
-namespace FakeItEasy.Tests.Configuration
-{
-    using System;
-    using System.Linq.Expressions;
-    using FakeItEasy.Configuration;
-    using FakeItEasy.Core;
-    using FakeItEasy.Expressions;
-    using FakeItEasy.Tests;
-    using FakeItEasy.Tests.TestHelpers;
-    using FluentAssertions;
-    using Xunit;
-
-    public class StartConfigurationTests

```

Comment:
I'd seen the class before, when trying to remove unused classes pre-2.0. But I didn't really understand what it did. I may not still.